db/seeds.rb

```diff
@@ -88,18 +151,35 @@
 
 bundle = person_collection.zip room_list
 
+point_list.each do |point|
+  Point.create(point)
+end
+
+point_of_interest_list.each do |point_of_interest|
+  PointOfInterest.create(point_of_interest)
+end
+
+polyline_list.each do |polyline_points|
+  Polyline.create(points: polyline_points.map { |point_id| Point.find(point_id) })
+end
+
 bundle.each do |person, room|
   Room.create(number: room[0], floor: room[1], full_name: room[2], people: [person])
 end
 
+hs_rooms.each do |room|
+  outer_shape = Polyline.find(room[2]) if room[2].present?
+  Room.create(floor: room[0], full_name: room[1], outer_shape: outer_shape)
+end
+
 room_type_list.each do |room_type|
   RoomType.create(name: room_type[0])
 end
```

Comment:
```suggestion
point_list.each do |point|
  Point.create(point)
end

point_of_interest_list.each do |point_of_interest|
  PointOfInterest.create(point_of_interest)
end

polyline_list.each do |polyline_points|
  Polyline.create(points: polyline_points.map { |point_id| Point.find(point_id) })
end

bundle = person_collection.zip hs_rooms

bundle.each do |person, room|
  outer_shape = Polyline.find(room[2]) if room[2].present?
  Room.create(number: room[0], floor: room[1], full_name: room[2], people: [person])
end

room_type_list.each do |room_type|
  RoomType.create(name: room_type[0])
end
```

Reply:
Fixed this not to use rooms_list anymore (use hs_rooms instead)

Am not sure about lines 166 through 169, looks to me like the same room may be created several times (not sure though)